src/amo/pages/LanguageTools/index.js

```diff
@@ -203,7 +203,7 @@ export class LanguageToolsBase extends React.Component<Props> {
                         return addon.target_locale === language.locale;
                       }
 
-                      const re = new RegExp(`^${language.locale}(-\\w+)?$`);
+                      const re = new RegExp(`^${language.locale}(-\\w+){0,2}$`);
```

Comment:
seems reasonable but do we need the regex at all?

Reply:
I think we do because we want to group addons belonging to the same short locale when we don't support the longer version of a locale, e.g., `fa` and `fa-IR` are both supported and so we have 2 different rows on the page. For `az` and `az-IR`, we only support `az` so we want both add-ons to be displayed on the single row (for `az`).